packages/clerk-js/src/core/clerk.ts

```diff
@@ -137,6 +144,14 @@ export default class Clerk implements ClerkInterface {
 
     this.proxyUrl = (options as ClerkConstructorOptions | undefined)?.proxyUrl;
 
+    if (this.proxyUrl) {
+      if (isValidProxyUrl(this.proxyUrl)) {
+        this.proxyUrl = isProxyUrlRelative(this.proxyUrl) ? window.location.origin + this.proxyUrl : this.proxyUrl;
```

Comment:
:wrench: Is it safe to join URL paths with string concatenation? Is there a chance that `window.location.origin` might have a trailing slash?

How about using `new URL(path, base).toString()` instead?